index.d.ts

```diff
@@ -215,7 +207,7 @@ declare class EthereumEvent {
   address: Address
   eventSignature: string
   blockHash: H256
-  params: Array<EthereumEventParam>
+  parameters: Array<EthereumEventParam>
```

Comment:
I'm ok with renaming this field but it seems unnecessary.

Reply:
The purpose is simply to match what's on `index.ts`